rules/windows/privilege_escalation_uac_bypass_com_interface_icmluautil.toml

```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/10/19"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies User Account Control (UAC) bypass attempts via the ICMLuaUtil Elevated COM interface. Attackers may attempt
+to bypass UAC to stealthily execute code with elevated permissions.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "UAC Bypass via ICMLuaUtil Elevated COM Interface"
+risk_score = 73
+rule_id = "68d56fdc-7ffa-4419-8e95-81641bd6f845"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ process.parent.name == "dllhost.exe" and
+ wildcard(process.parent.args,"/Processid:{3E5FC7F9-9A51-4367-9063-A120244FBEC7}","/Processid:{D2E7041B-2927-42FB-8E9F-7CE93B6DC937}") and
```

Comment:
since there are no wildcards and its an exact match, we don't really need the `wildcard` function. i'm not sure it hurts anything though(?), or if `process.parent.args in (...)`  or `process.parent.args : (...)` would be more optimal, the latter of which would be case insensitive going forward? or `wildcard~(...)`? maybe @rw-access can weigh in.

Reply:
otherwise this looks good to me

Reply:
yeah, I think we should switch from `wildcard()` to `in` in this case.
Elasticsearch will detect if there are no wildcards, so they should have the same performance ultimately anyway. But it's good to be consistent

Reply:
changed